.github/workflows/apt.yml

```diff
@@ -8,11 +8,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
-          - debian-buster
-          - ubuntu-xenial
-          - ubuntu-bionic
-          - ubuntu-focal
+        label:
+          - Debian GNU/Linux Buster amd64
+          - Ubuntu Xenial amd64
+          - Ubuntu Bionic amd64
+          - Ubuntu Focal amd64
+        include:
+          - label: Debian GNU/Linux Buster amd64
+            rake-job: debian-buster
+            test-docker-image: debian:buster
+            test-script: td-agent/apt/test.sh
```

Comment:
It seems that it is better to use meaningful file name.

For example. install-test.sh and so on.

Reply:
Fixed in https://github.com/fluent-plugins-nursery/td-agent-builder/pull/141/commits/db688448a4b1e6ecfbefe3555754ee70f509ccd4.